### DISCUSSION TITLE
Shaper calibration

### DISCUSSION THREAD

**Discussion by lvlacleod** [Guilouz/Creality-Helper-Script-Wiki#485] [Issues]
This line near the bottom seems to delete the csv after it is created making it impossible to download. Am I missing something for a way to view the graphs? It is also present in test resonance and belt shaper

`  RUN_SHELL_COMMAND CMD=delete_csv`

I had to commend it out to get the csv's

**Comment by TheWebMachine** [comment]
It uses the CVSs to generate PNG graphs, if I'm not mistaken.

**Reply by lvlacleod** [reply]
So where are the PNG's saved because the csv's used to be saved in /tmp/? It never used to generate PNG's for input shaper.

**Comment by Guilouz** [comment]
The CSV is of no use as such, it is just used to generate the graphs and I delete them to prevent the graphs from being generated from an old CSV.

**Reply by lvlacleod** [reply]
Where are the graphs saved. The script used to save the csv's to /tmp/. I cannot find any csv's or png's after the input shaper macro is finished

**Comment by Guilouz** [comment]
INPUT_SHAPER_CALIBRATION Macro just start calibration and save settings and not generate graphs.

TEST_RESONANCES_GRAPHS Macro is used to mesure and generate graphs in Helper-Script folder.

**Reply by TheWebMachine** [reply]
That's it! I forgot which one generated graphs and which one didn't.

**Comment by JULIANWAT** [comment]
I am a newby and have a K1C. When you run the BELTS_SHAPER_CALIBRATION  you get a graph. As I SEE IT, there are two aspects of the graph. The first is the tensions on the two belts are the same hence similar peak frequencies and "simalarity" arround 95% . The second is the actual peak frequency , I ASSUME reflects the tension in the belts. WHAT SHOULD THAT FREQUENCY  BE ??
